packages/mandelbrot/src/filters.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (theme, env, app) {
     });
 
     env.engine.addFilter('linkRefs', function (str, item) {
-        if (!(item.isComponent || item.isVariant)) {
+        if (!(item.isComponent || item.isVariant) || !item.references.length) {
```

Comment:
Not sure what other types of items can be passed in here besides components/variants, but if `item.isComponent === false` and `item.isVariant === false`, then if `item.references === undefined`, this will error out. Maybe do a second if with an early return instead?

Or... maybe I'm reading this wrong because the condition is a bit complex :( I think even if my previous explanation is not true, a second if clause with an early return would improve readability.

Reply:
Done. It's functionally equivalent to the previous commit, but indeed more readable 🙂

Reply:
Conditions with multiple or's and negations make my brain hurt :) Much better now.